src/app/address/controllers/addressConfirm.js

```diff
@@ -15,47 +15,44 @@ const {
 class AddressConfirmController extends BaseController {
   locals(req, res, callback) {
     super.locals(req, res, (err, locals) => {
-      const sessionAddresses = req.sessionModel.get("addresses");
+      const currentAddress = req.sessionModel.get("address");
+      const previousAddress = req.session["hmpo-wizard-previous"]?.address;
```

Comment:
Accessing the alternative wizard like this for now.

In the future we could make the confirmation step a third wizard that orchestrates the others?
As in the future we may wish to get more than one previous address. Food for thought.

Reply:
Yeah, that would be my thought - two wizards `current` and `previous` using mostly the same config, feeding into a `summary` one. 

This is how the summary screen on the HMPO journey works, feeding back into other wizards for editing as needed, invalidating fields if values changes (e.g. if year changes to 2000, then the previous address data is now invalid).